charts/mlflow/templates/artifacts-deployment.yaml

```
@@ -0,0 +1,245 @@
+          {{- end }}
+          resources:
+            requests:
+              cpu: 5m
+              memory: 8Mi
+            limits:
+              cpu: 50m
+              memory: 32Mi
+        {{- end }}
+{{- end }}
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (2 > 0)

(empty-lines)